packages/core/src/App/Containers/OneTimeDepositModal/one-time-deposit-modal.tsx

```diff
@@ -29,15 +26,22 @@ const OneTimeDepositModal = observer(() => {
     } = ui;
     const liveChat = useLiveChat(false, loginid);
 
+    /** fetch account_status every 2 seconds since we can't subscribe for status updates */
     React.useEffect(() => {
-        setIsIframeLoading(true);
-    }, [iframe_url]);
+        const interval = setInterval(() => {
+            updateAccountStatus();
+        }, 2000);
```

Comment:
Since we cant subsribe to account_status and we rely on status (based on BE changes) to know if user has deposited, so i fetch account_status every 2 seconds here